internal/driver/common.go

```
@@ -0,0 +1,104 @@
+package driver
+
+import (
+	"fmt"
+
+	"github.com/rayakame/sqlc-gen-better-python/internal/config"
+	"github.com/rayakame/sqlc-gen-better-python/internal/model"
+	"github.com/rayakame/sqlc-gen-better-python/internal/writer"
+	"github.com/sqlc-dev/plugin-sdk-go/metadata"
+)
+
+func writeFuncSignature(body *writer.CodeWriter, d Driver, config *config.Config, indent int, query model.Query, returnAnnotation string) string {
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(golines)